grafana/scylla-dash.master.template.json

```diff
@@ -77,7 +77,8 @@
                                 "step": 4
                             }
                         ],
-                        "title": "Load"
+                        "title": "Load",
+                        "description": "the percentage of the time during which the CPU is utilized by Scylla. Note that because Scylla does busy polling for some time before going idle, CPU utilization as seen by the operating system may be much higher. Your system is not yet CPU-bottlenecked until this metric is high"
```

Comment:
Some of the descriptions are capitalized, others are not, some ends with a period, others are not, choose one.
Maybe switch to during Scylla utilized the CPI.